src/helpers/config.ts

```diff
@@ -439,13 +447,23 @@ export const allCoinMetas: Record<string, CoinMetadata> = {
     Description: 'Polygon Mumbai Testnet',
     Icon: 'polygon',
     value: 'tpolygon',
+    isTssSupported: true,
+    ApiKeyProvider: 'api-amoy.polygonscan.com',
+  },
+  tbsc: {
+    Title: 'TBSC',
+    Description: 'BNB Smart Chain Testnet',
+    Icon: 'bsc',
+    value: 'tbsc',
+    isTssSupported: true,
+    ApiKeyProvider: 'api-amoy.polygonscan.com',
```

Comment:
Not sure if this is the right provider for bsc

Reply:
I think it is not being used, removing it